src/components/MultiHopTrade/components/TradeInput/TradeInput.tsx

```diff
@@ -78,45 +84,45 @@ import {
 } from 'state/slices/tradeQuoteSlice/selectors'
 import { tradeQuoteSlice } from 'state/slices/tradeQuoteSlice/tradeQuoteSlice'
 import { useAppDispatch, useAppSelector } from 'state/store'
+import { breakpoints } from 'theme/theme'
 
 import { useAccountIds } from '../../hooks/useAccountIds'
 import { useSupportedAssets } from '../../hooks/useSupportedAssets'
-import { PriceImpact } from '../PriceImpact'
+import { QuoteList } from '../QuoteList/QuoteList'
+import { HorizontalCollapse } from './components/HorizontalCollapse'
 import { RecipientAddress } from './components/RecipientAddress'
 import { SellAssetInput } from './components/SellAssetInput'
-import { TradeQuotes } from './components/TradeQuotes/TradeQuotes'
+import { CountdownSpinner } from './components/TradeQuotes/components/CountdownSpinner'
 import { getQuoteErrorTranslation } from './getQuoteErrorTranslation'
 import { getQuoteRequestErrorTranslation } from './getQuoteRequestErrorTranslation'
+import { useSharedHeight } from './hooks/useSharedHieght'
 
 const formControlProps = {
   borderRadius: 0,
   background: 'transparent',
   borderWidth: 0,
-  paddingBottom: 0,
 }
-
 const arrowDownIcon = <ArrowDownIcon />
 const emptyPercentOptions: number[] = []
 
-export const TradeInput = memo(() => {
-  const {
-    isAnySwapperFetched: _isAnySwapperFetched,
-    isQuoteRequestComplete: _isQuoteRequestComplete,
-    isQuoteRequestUninitialized,
-    didQuoteRequestFail,
-    isQuoteRequestIncomplete,
-  } = useGetTradeQuotes()
+type TradeInputProps = {
+  isCompact?: boolean
+}
+
+export const TradeInput = memo(({ isCompact }: TradeInputProps) => {
+  useGetTradeQuotes()
   const {
     state: { wallet },
   } = useWallet()
+  const { observedRef: tradeInputRef, height: tradeInputHeight } = useSharedHeight()
+  const [isSmallerThanXl] = useMediaQuery(`(max-width: ${breakpoints.xl})`, { ssr: false })
   const { handleSubmit } = useFormContext()
   const dispatch = useAppDispatch()
   const mixpanel = getMixPanel()
   const history = useHistory()
   const { showErrorToast } = useErrorHandler()
+  const [isCompactQuoteListOpen, setIsCompactQuoteListOpen] = useState(false)
```

Comment:
`useToggle` ?

Reply:
not on this one, we need explicit set true/false in this case